app/assets/stylesheets/dm/components/_links.scss

```diff
@@ -17,6 +17,10 @@
   }
 }
 
+.internal-server-error-link{
```

Comment:
we don't need this new class we can use `usa-link` inline instead

Reply:
Got it.